### PR TITLE
fix: Make keeper image pull policy configuration consistent

### LINKED ISSUE #982: Any reason why keeper.imagePullPolicy is not made on the same pattern as others ?
https://github.com/jenkins-x/lighthouse/blob/53f2eae69b75b62d1b6e2a4910e8a3dd8072d63f/charts/lighthouse/values.yaml#L163

For example, foghorn (and all other elements in the chart) uses `keeper.image.pullPolicy`.

### PR DESCRIPTION
**Pull request by abayer**
fixes #982

Signed-off-by: Andrew Bayer <andrew.bayer@gmail.com>

### CODE REVIEW

**Comment by abayer** [comment]
@eddycharly Beat you to it. =)

**Comment by eddycharly** [comment]
Ahah, how about the doc ?

**Comment by abayer** [comment]
whoops!

File: charts/lighthouse/values.yaml [hunk 1]
```diff
@@ -160,7 +160,7 @@ keeper:
   image:
     repository: "{{ .Values.image.parentRepository }}/lighthouse-keeper"
     tag: "{{ .Values.image.tag }}"
-  imagePullPolicy: IfNotPresent
+    pullPolicy: "{{ .Values.image.pullPolicy }}"
```
Comment (eddycharly) [inline charts/lighthouse/values.yaml:163]:
```suggestion
    # keeper.image.pullPolicy -- Template for computing the keeper controller docker image pull policy
    pullPolicy: "{{ .Values.image.pullPolicy }}"
```

Comment (abayer) [reply charts/lighthouse/values.yaml:163]: Adding now.

File: charts/lighthouse/config/lighthousejobs.lighthouse.jenkins.io.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```
Comment (abayer) [inline charts/lighthouse/config/lighthousejobs.lighthouse.jenkins.io.yaml:1]: Running locally I got a validation error on `apiextensions.k8s.io/v1beta`, so hey, let's switch to `v1`.

**Comment by abayer** [comment]
Well, boo:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(CustomResourceDefinition.spec): unknown field "subresources" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceDefinitionSpec, ValidationError(CustomResourceDefinition.spec): unknown field "version" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceDefinitionSpec, ValidationError(CustomResourceDefinition.spec): missing required field "versions" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceDefinitionSpec]
```
Reworking the CRD now. =)

**Comment by eddycharly** [comment]
Hmmm.. i can take care of it if it's too much trouble ;)
We can upgrade the CRD in another PR.

**Comment by abayer** [comment]
I think I got it. =)

**Comment by abayer** [comment]
I might add the schema in a followup.

**Comment by abayer** [comment]
blech, schema is required! Guess I'll get that figured out, since I was looking for something to do today... =)

**Comment by abayer** [comment]
Ok, it's a hefty enough task that I'm going to separate that out into a new issue/PR and just revert back the CRD version in this PR.

**Comment by eddycharly** [comment]
Fair enough, if you want i can take care of the image pull policy and you battle with the CRD ?

**Comment by eddycharly** [comment]
It will be easier to get the pull policy in as you can approve it =)

**Comment by abayer** [comment]
This should be good now, and I can `/approve` if you `/lgtm` anyway. =)

**Pull request merged**